app/controllers/publishers/organisations/managed_organisations_controller.rb

```diff
@@ -9,7 +9,7 @@ def show
   end
 
   def update
-    @managed_organisations_form = ManagedOrganisationsForm.new(managed_organisations_params)
+    @managed_organisations_form = ManagedOrganisationsForm.new(params[:managed_organisations_form][:sort_column], managed_organisations_params)
```

Comment:
This controller is only for the managed organisations page, so sorting here is never applicable